Source/dom/components/ImportableObject.js

```diff
@@ -109,11 +111,10 @@ ImportableObject.define('objectType', {
   exportable: true,
   importable: false,
   get() {
-    return ImportableObjectType.Symbol
```

Comment:
What do you think about having a method to retrieve the SymbolMaster if the importable object is a symbol? I can't find an easy way to get the Symbol from the importable object.